src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -195,7 +206,12 @@ public void testWithCacheWithNoViolation() throws Exception {
         final File file = temporaryFolder.newFile("file.java");
         final List<String> lines = new ArrayList<>();
         lines.add(" class a {} ");
-        treeWalker.processFiltered(file, lines);
+        try {
+            treeWalker.processFiltered(file, lines);
+        }
+        catch (CheckstyleException ex) {
+            fail("Exception is not expected");
```

Comment:
@rnveach, francly , all this "tets" were testing nothing, only rasing up coverage by calling not covered methods, I don't understand the purpose of such tests. @romani said I should add the most simple to check asserts. If it is not ok for now, I will rewrite.